modules/programs/alacritty.nix

```diff
@@ -41,10 +41,13 @@ in
     };
   };
 
-  config = mkIf cfg.enable {
-    home.packages = [ pkgs.alacritty ];
-
-    xdg.configFile."alacritty/alacritty.yml".text =
-      replaceStrings ["\\\\"] ["\\"] (builtins.toJSON cfg.settings);
-  };
+  config = mkMerge [
+    (mkIf cfg.enable {
+      home.packages = [ pkgs.alacritty ];
+    })
+    (mkIf (cfg.settings != {}) {
```

Comment:
This means that the settings will be written if `enable` is false and `settings` is not empty.

Reply:
Yes, @kalbasit is right. I would suggest having something like
```
config = mkIf cfg.enable {
  home.packages = [ pkgs.alacritty ];

  xdg.configFile."alacritty/alacritty.yml" = mkIf (cfg.settings != {}) {
    text = replaceStrings ["\\\\"] ["\\"] (builtins.toJSON cfg.settings);
  };
};
```
instead.

Reply:
Fixed.